lunatrace/bsl/hasura/migrations/lunatrace/1672788403469_add_cisa_known_exploited_vulnerabilities/up.sql

```diff
@@ -0,0 +1,25 @@
+-- Indexes to speed up EPSS inserter
+CREATE INDEX IF NOT EXISTS vulnerability_equivalent_b_idx ON vulnerability.equivalent (b);
+
+CREATE INDEX IF NOT EXISTS vulnerability_vulnerability_source_id_idx ON vulnerability.vulnerability (source_id);
+
+-- Table to hold the CISA Known Exploited vulnerabilities
+CREATE TABLE IF NOT EXISTS vulnerability.cisa_known_exploited_vulnerabilities (
+    cve TEXT PRIMARY KEY,
+    vendor_project text NOT NULL,
+    product text NOT NULL,
+    vulnerability_name text NOT NULL,   
+    date_added date NOT NULL,
+    short_description text NOT NULL,
+    required_action text NOT NULL,
+    due_date date NOT NULL,
+    notes text NOT NULL,
+    CHECK (cve LIKE 'CVE-%')
+);
+
+-- Reference the CISA Known Exploited vulnerabilities table by CVE
+ALTER TABLE vulnerability.vulnerability
+    ADD COLUMN IF NOT EXISTS cisa_known_exploited_cve TEXT
```

Comment:
this column will always contain the cve id again? should we make this a backref?